Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -566,8 +566,8 @@ public final class ManifestLoader: ManifestLoaderProtocol {
 
             try withTemporaryDirectory(removeTreeOnDeinit: true) { tmpDir in
                 // Set path to compiled manifest executable.
-                let file = tmpDir.appending(components: "\(packageIdentity)-manifest")
-                cmd += ["-o", file.pathString]
+                let compiledManifestFile = tmpDir.appending(component: "\(packageIdentity)-manifest")
```

Comment:
Clarified the name here, since there is now more than one file being dealt with in this code block.